tasks/eth/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,123 @@
+  **After**: `0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375`
+  **Meaning**: Updates the implementation address of the Proxy to the standard SystemConfig implementation.
+
+### `0xA3cAB0126d5F504B071b81a3e8A2BBBF17930d86` (`SystemConfigProxy` for zora-mainnet)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+  **After**: `0x00000000000000000000000000000000000941ad000003f40000000001c9c380`
+  **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `100000` and `611590` respectively. These share a slot with the `gasLimit` which remans at `30000000`
+
+- **Key**: `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  **Before**: `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1`
+  **After**: `0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375`
+  **Meaning**: Updates the implementation address of the Proxy to the standard SystemConfig implementation.
+
+### `0x34A564BbD863C4bf73Eca711Cf38a77C4Ccbdd6A` (`SystemConfigProxy` for arena-z-mainnet)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000066`
+  **Before**: `0x00000000000000000000000000000000000000000000000000000000000a6fe0`
+  **After**: `0x01000000000000000000000000000000000000000000000000000000000a6fe0`
+  **Meaning**: Updates the `scalar` storage variable to reflect a scalar version of `1`.
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000003938700`
+  **After**: `0x0000000000000000000000000000000000000000000c3c9d0000000003938700`
+  **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `801949` and `0` respectively. These share a slot with the `gasLimit` which remains at `60000000`
+
+- **Key**: `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  **Before**: `0x000000000000000000000000f56d96b2535b932656d3c04ebf51babff241d886`
+  **After**: `0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375`
+  **Meaning**: Updates the implementation address of the Proxy to the standard SystemConfig implementation.
+
+
+### `0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A` (`ProxyAdminOwner` for all chains in this task)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000007`
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000008`
+  **Meaning**: Nonce increments by 1
+
+- **Key**: See above.
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000001`
+  **Meaning**: approvedHashes update. See above.
+
+
```

Comment:
You will probably be asked to also provide a list of all expected nonce changes, like for eth/026.

Reply:
[a0215e9](https://github.com/ethereum-optimism/superchain-ops/pull/454/commits/a0215e913450bdc943da38ad19d84e8e283a4c17)